site/src/pages/release-notes/1.13.3.mdx

```diff
@@ -0,0 +1,53 @@
+---
+date: 2021-11-23
+---
+
+## 🌟 New features
+
+- N/A
+
+## 📈 Improvements
+
+- N/A
+
+## 🛠️ Bug fixes
+
+- <type://DecodingClient> does not raise an exception anymore when duplicated encodings are specified. #3932 #3934
+  - The duplicated encodings are silently ignored.
+- `unfinishedResponse` is correctly decremented when the <type://HttpResponse> is cancelled. #3920
+- The text in the description sections are now correctly rendered in DocService. #3917
+- The status of a protocol violation is now correctly logged by <type://LoggingService> #3914
+    - A response headers is exactly written once when a protocol violation error is raised.
+- `GrpcDocStringExtractor` now reads description from `bin`, `desc`, `dsc`, `pb`, and `protobin` file. #3912
```

Comment:
```suggestion
- <type://GrpcDocServicePlugin> now reads description from `bin`, `desc`, `dsc`, `pb`, and `protobin` file. #3912
```

Reply:
`GrpcDocServicePlugin` is in an internal package so let me use 
`<type://DocServicePlugin> for gRPC` 😉

Reply:
> GrpcDocServicePlugin is in an internal package

Oops... 😅